spec/controllers/admin/reports_controller_spec.rb

```diff
@@ -352,4 +352,28 @@
       end
     end
   end
+
+  context "Sales Tax Reports By Order" do
+    let!(:present_objects) { [orderA1, orderA2, orderB1, orderB2] }
+    let(:report_type) { :sales_tax_totals_by_order }
+    context "as an admin" do
+      before do
+        controller_login_as_admin
+      end
+      it "generates the report" do
```

Comment:
@dacook mentioned it somewhere else, but can we have spaces between block here and below as well. I am not sure if it should block the PR so leaving it as a comment.